resources/k8s-artifacts/choreo-connect/adapter-deployment.yaml

```diff
@@ -66,13 +66,13 @@ spec:
             - containerPort: 18000
             - containerPort: 9843
           livenessProbe:
-            tcpSocket:
-              port: 9843
+            exec:
+              command: [ "/bin/grpc_health_probe", "-addr", "localhost:18000", "-tls", "-tls-ca-cert", "/home/wso2/security/keystore/mg.pem", "-tls-client-cert", "/home/wso2/security/keystore/mg.pem", "-tls-client-key", "/home/wso2/security/keystore/mg.key" ]
```

Comment:
@pubudu538 @Rajith90 shall we introduce ENV var for this cert path same as for the Adapter?
Or shall we introduce a script?

Reply:
Add a script to execute this

Reply:
Make timeout to 3seconds and Add Env var for certs

Reply:
Added a script to execute health probe, with setting the timeout to 3s and added env var for cert in the adapter with https://github.com/wso2/product-microgateway/pull/1899/commits/f0a5fadbd9557078c502c8ae78f371c38732e871